### PR TITLE
First test Geico

### PR DESCRIPTION
**Pull request by Elena-Shagdurova**

### CODE REVIEW

File: src/test/java/school/redrover/GroupUnicornsTest.java [hunk 1]
```diff
@@ -58,4 +61,25 @@ public void testLoginAttemptWithInvalidUsername() {
...
         driver.quit();
     }
+
+    @Test
+    public void testGeico() {
+        WebDriver driver = new ChromeDriver();
+        try {
+            driver.get("https://www.geico.com/");
+
+            WebElement title = driver.findElement(By.xpath("//div/h1[@id ='section1heading']"));
```
Comment (taysan003) [inline src/test/java/school/redrover/GroupUnicornsTest.java:71]: it's better to use driver.findElement(By.id("section1heading")0:

File: src/test/java/school/redrover/GroupUnicornsTest.java [hunk 2]
```diff
@@ -58,4 +61,25 @@ public void testLoginAttemptWithInvalidUsername() {
...
+            title.isDisplayed();
+
+            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(1000));
+
+            WebElement zipCode = driver.findElement(By.xpath("//div/input[@id ='ssp-service-zip']"));
```
Comment (taysan003) [inline src/test/java/school/redrover/GroupUnicornsTest.java:76]: it's better to use id  driver.findElement(By.id("ssp-service-zip"));

Comment (taysan003) [reply src/test/java/school/redrover/GroupUnicornsTest.java:76]: ID is most likely will be unique element on the page and it means more reliable

**Comment by Elena-Shagdurova** [comment]
I got you. Thank you for your feedback.

**Review by PeterKuzyk — APPROVED** [review]

**Review by SafronovaOlga — APPROVED** [review]

File: src/test/java/school/redrover/GroupUnicornsTest.java [hunk 3]
```diff
@@ -58,4 +61,25 @@ public void testLoginAttemptWithInvalidUsername() {
...
+            zipCode.sendKeys("11111");
+
+            WebElement submit = driver.findElement(By.xpath("//input[@class ='btn btn--secondary']"));
+            submit.click();
```
Comment (bzhumanova) [inline src/test/java/school/redrover/GroupUnicornsTest.java:80]: don't you want to check the message after clicking on the Go button "Thanks! Select insurance products below to start a free quote:"?

**Review by bzhumanova — APPROVED** [review]

**Review by Yauheni95 — APPROVED** [review]

File: src/test/java/school/redrover/GroupUnicornsTest.java (hunk 3 above)
Comment (Elena-Shagdurova) [reply src/test/java/school/redrover/GroupUnicornsTest.java:80]: Thank you. I'll do later

**Pull request closed without merging**